pkg/cmd/pulumi/login.go

```diff
@@ -120,6 +120,10 @@ func newLoginCmd() *cobra.Command {
 				if err != nil {
 					return fmt.Errorf("could not determine current cloud: %w", err)
 				}
+			} else if url := strings.TrimPrefix(strings.TrimPrefix(
+				cloudURL, "https://"), "http://"); strings.HasPrefix(url, "app.pulumi.com/") {
```

Comment:
Wonder if we should just do contains check for pulumi.com? Would also catch if people tried to do like `pulumi login pulumi.com`